packages/Geographic/README.md

```diff
@@ -0,0 +1,51 @@
+## iTowns Geographic
+
+The geographic package provides utilities for handling coordinates, ellipsoids, extents and rotations across different coordinate systems.
+
+* [Coordinates](http://www.itowns-project.org/itowns/docs/#api/Geographic/Coordinates) : A Coordinates object (geodetic datum), defined by a [crs] and three values.
+* [Extent](http://www.itowns-project.org/itowns/docs/#api/Geographic/Extent) : Extent is geographical bounding rectangle defined by 4 limits: west, east, south and north.
+* [Crs](http://www.itowns-project.org/itowns/docs/#api/Geographic/CRS) : This module provides basic methods to manipulate a CRS (as a string). Visiting [epsg.io](https://epsg.io/) to more coordinate system worldwide. Use **Proj4js** definition in epsg.io.
+* [OrientationUtils](http://www.itowns-project.org/itowns/docs/#api/Geographic/OrientationUtils) : it provides methods to compute the quaternion that models a rotation defined with various conventions, including between different CRS.
+* Ellipsoid : In geodesy, a reference ellipsoid is a mathematically defined surface that approximates the geoid.
```

Comment:
I'm not sure we should add all classes here as they already each have their own doc? WDYT @Desplandis? If we keep this list, we should add all classes though

Reply:
I think it's a good idea to introduce the module's features and point to the detailed documentation. 

 > If we keep this list, we should add all classes though

Except for the Main sub-module, which has yet to be divided.

Reply:
For such a small package, we can explicit the different kind of modules. But I don't think this could scale-up to other big packages.